src/app/media-wall/media-wall-design/media-wall-design.component.scss

```diff
@@ -87,3 +87,31 @@ span.apply-button {
 		}
 	}
 }
+
+#media_wd_heading {
+	font-size: 30px;
+}
+
+#selectedFeedsId {
+	margin-top:8px;
+	margin-left:30px;
+	width: 20%;
+}
+
+#selectedCardId {
+	margin-top:20px;
+}
+
+#selectedColumnId {
+	margin-top:10px;
+	width: 50%;
+}
+
+#count1 {
+	margin-right: 20px;
+	width: 40%;
+}
+
+#card_style {
+	 margin-top: 12px;
+}
```

Comment:
Insert a new line at the end.

Reply:
![screen shot 2018-03-18 at 11 01 21 pm](https://user-images.githubusercontent.com/26062692/37568936-056306f2-2b02-11e8-8f31-92b2dc4b4d4f.png)
Apparently the Travis failed because of this error.

Reply:
I'll still insert a newline and make a commit 😄

Reply:
@Remorax New line is not inserted yet at the end. Please make the changes.